test/liquid-raw-tag/index.liquid

```diff
@@ -21,6 +21,25 @@ It should not reindent raw tags, as this tag is indentation sensitive
 {% endraw %}
 </div>
 
+It should be indentation (and whitespace) sensitive
```

Comment:
I'm not sure I understand 😓. Is this about the lack of whitespace around the tag? 

The example might be a bad one because I chose the `p` tag which is leading and trailing inner whitespace insensitive. What I'm concerned about here is whatever is _inside_ `raw` and not really what is around it. The `<p>` tag was there to force the formatter to _try_ to reindent it. 

So, 
```liquid
{% # input %}
<p>{% raw %}
  # markdown example

  - this is a list
        * this is indented with mixed tabs!
{% endraw %}</p>

{% # expected output %}
<p>
  {% raw %}
  # markdown example

  - this is a list
        * this is indented with mixed tabs!
{% endraw %}
</p>
```

Which, granted, looks kind of odd. But the following are all ok:
- All whitespace _inside_ the raw tag is maintained
- The leading whitespace is not meaningful, so :+1:
- The trailing whitespace is not meaningful, so :+1: